data/dataloader.py

```diff
@@ -87,7 +90,55 @@ def to_cuda(item, computing_device, label=False):
     else:
         item = Variable(item.to(computing_device)).float()
     return item
+    
 
+def create_class_dict (lbs_all_classes):
+    """
+    Helper Function: Create CLASSES Dictionary and assign in to global CLASSES vairable
+    """
+    global CLASSES
+    lbs_all_classes.sort()
+    CLASSES = {}
+    c = 0
+    for i in lbs_all_classes:
+        CLASSES[c] = i
+        c+=1
+    NUM_CLASSES = len(CLASSES.keys())
+    print('All classes detected are: '+str(CLASSES))
```

Comment:
Are we printing statements or using loggers. I remember using loggers in our code last quarter. Let's stick to one thing

Reply:
agree. I will modify the print to log